### DISCUSSION TITLE
C language

### DISCUSSION THREAD

**Discussion by ankit-akash** [ankit-akash/DSA#13] [Q&A] [answered]
How is an array initialized in C language? Write the correct option below.
a. int a[3]={1,2,3}
b. int a={1,2,3}
c. int a(3)={1,2,3}
d. int a[]=new int[3]

**Comment by Preetiraj3697** [comment, marked as answer]
I don't know C language. But I try I don't know my answer is right. But answer is A. int a[3] = {1,2,3}. I know JavaScript. JavaScript little bit similar answer that why this is my answer. I not sure 100% this answer is right. But 80% this is right answer.

**Reply by ankit-akash** [reply]
thats the right answer!
